packages/local_auth/ios/Classes/LocalAuthPlugin.m

```diff
@@ -62,6 +64,26 @@ - (void)alertMessage:(NSString *)message
                                                                                    completion:nil];
 }
 
+- (void)getAvailableBiometrics:(FlutterResult)result {
+  LAContext *context = [[LAContext alloc] init];
+  NSError *authError = nil;
```

Comment:
Collect the results in a list and return them like you do in java code rather than returning directly. See https://developer.apple.com/documentation/foundation/nsmutablearray .